problemsets-mat223/modules/module10.tex

```diff
@@ -35,12 +35,22 @@
 our methods will be ad hoc.
 
 \begin{example}
-	Let $\mathcal U:\R^2\to\R^2$ be the transformation induced by $M=\matc{\sqrt{2}/2&-\sqrt{2}/2\\0&0}$,
+	Let $\mathcal U:\R^2\to\R^2$ be the transformation given by $M=\matc{\sqrt{2}/2&-\sqrt{2}/2\\0&0}$,
 	let $\mathcal R:\R^2\to\R^2$ be rotation counter clockwise by $45^\circ$, and let $\mathcal P:\R^2\to\R^2$
 	be projection onto the $x$-axis.
 	Write $\mathcal U$ as the composition (in some order) of $\mathcal R$ and $\mathcal P$.
 
-	XXX Finish
+	For an arbitrary vector $\vec x=\mat{x\\y}=\mat{r\cos\theta\\r\sin\theta}$, where $r$ is the length of the vector. Notice that
```

Comment:
This is too complicated. Just transform `$\xhat$` and `$\yhat$`, which can be done by inspection.

Reply:
I have changed it into "By inspection".
But I will not be able to inspect it out without your hint. Is it really able to inspect? hahaha I feel it is really complicated for me to inspect

Reply:
I know that you can rotate the vector [1,0]  by 45 degrees by just drawing a picture ;-).